common/util.c

```diff
@@ -88,6 +88,9 @@ bool read_ed25519_key(uint8_t *in, size_t in_len, uint8_t *out,
   int ret;
   BIO *b64 = BIO_new(BIO_f_base64());
   BIO *bio = BIO_new(BIO_s_mem());
+  if (b64 == NULL || bio == NULL) {
```

Comment:
this may leak memory if allocation of b64 succeeds but bio fails (unlikely, yes.)